sorc/build_ufs.sh

```diff
@@ -7,15 +7,17 @@ cwd=$(pwd)
 APP="S2SWA"
 CCPP_SUITES="FV3_GFS_v17_p8_ugwpv1,FV3_GFS_v17_coupled_p8_ugwpv1,FV3_global_nest_v1"  # TODO: does the g-w need to build with all these CCPP_SUITES?
 PDLIB="ON"
+HYDRO="OFF"
 
-while getopts ":da:fj:vw" option; do
+while getopts ":da:fj:vwy" option; do
   case "${option}" in
     d) BUILD_TYPE="Debug";;
     a) APP="${OPTARG}";;
     f) FASTER="ON";;
     j) BUILD_JOBS="${OPTARG}";;
     v) export BUILD_VERBOSE="YES";;
     w) PDLIB="OFF";;
+    y) HYDRO="ON";;
```

Comment:
Is there a need to create a separate executable, or can we run hydrostatic mode with the non-hydrostatic executable?

Reply:
@junwang-noaa might know.

Reply:
Yes, you do need to compile with model with "HYDRO" turned on for the SFS case.

Reply:
Okay, thanks @junwang-noaa